tests/filecheck/dialects/qssa/ops.mlir

```diff
@@ -0,0 +1,28 @@
+// RUN: XDSL_ROUNDTRIP
+// RUN: XDSL_GENERIC_ROUNDTRIP
+
+%q0, %q1 = qssa.alloc<2>
+
+// CHECK: %q0, %q1 = qssa.alloc<2>
+
+%0 = qssa.h %q0
+
+// CHECK-NEXT: %0 = qssa.h %q0
+
+%q2:2 = qssa.cz %q1 %0
+
+// CHECK-NEXT: %q2, %q2_1 = qssa.cz %q1 %0
+
+%q3, %q4 = qssa.cnot %q2#0 %q2#1
```

Comment:
Just to make sure, you don't mean the notation of `%q2#0 %q2#1`? Because that's just a really weird alternative way for MLIR to represent multiple return values

Reply:
`%q2:2 = ...` is the same as saying `%q2_1, %q2_2 = ...`, just that the two ssa vals are accessed with `%q2#0` and `%q2#1` instead.

Reply:
Yeah that's actually what I was confused about, it looked like an array.